front/app/modules/commercial/content_builder/services/contentBuilder.ts

```diff
@@ -13,8 +16,6 @@ export interface IContentBuilderLayoutData {
     craftjs_jsonmultiloc: JsonMultiloc;
     code: string;
     enabled: boolean;
-    created_at: string;
-    updated_at: string;
   };
```

Comment:
Just curious why we can remove these? If they aren't relevant to the frontend, can we just leave them out like this (so our IContentBuilderLayoutData doesn't need to be a 1-to-1 match to the response from the backend)?

Reply:
It's a good practice to only type the attributes that you need in the code - keeps it cleaner. If we ever need this in the future - we can always add it 👍